libs/ledger-live-common/src/exchange/swap/index.ts

```diff
@@ -17,10 +17,11 @@ import {
   ValidationError,
 } from "../../errors";
 import getCompleteSwapHistory from "./getCompleteSwapHistory";
-import getExchangeRates from "./getExchangeRates";
-import getProviders from "./getProviders";
 import initSwap from "./initSwap";
 import { postSwapAccepted, postSwapCancelled } from "./postSwapState";
+import getExchangeRates from "./getExchangeRates";
+import getProviders from "./getProviders";
+import { isIntegrationTestEnv } from "./utils/isIntegrationTestEnv";
```

Comment:
Not now, but when we move this code to web....I think we must try avoid use "isIntegrationTestEnv" in the logic, we could mock the calls or run a fake server...what do you think?

Reply:
As you said, one for the future, but I'd recommend this when we move to a live app for swap. https://mswjs.io/